Server/EncryptionService/redis_logger.py

```
@@ -0,0 +1,379 @@
+                decode_responses=True,
+                ssl_cert_reqs=None,  # Disable SSL certificate verification
+                ssl_check_hostname=False,  # Disable hostname verification
+                socket_connect_timeout=5,  # 5 second connection timeout
+                socket_timeout=5,  # 5 second socket timeout
+                retry_on_timeout=True,
+                health_check_interval=30
+            )
+            self.redis_client.ping()
+            logger.info("Redis logger connection established successfully")
+
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)